allennlp/tests/training/trainer_test.py

```diff
@@ -1001,23 +1000,3 @@ def test_trainer_can_run_amp(self):
             opt_level="O1",
         )
         _ = trainer.train()
-
-
-class TestSparseClipGrad(AllenNlpTestCase):
```

Comment:
Does this test actually pass with the torch gradient clipping? The reason we implemented it ourselves is so that it works for sparse embedding layers - looking at the pytorch docs for `clip_grad_norm` i'm not sure that they've handled this case. Possibly it's fixed by them implementing `norm` for sparse tensors, but it doesn't look like they have from the docs: https://pytorch.org/docs/stable/sparse.html

Reply:
@bellecarrell could you put that test back with torch's `clip_grad_norm` substituted in?